### PR TITLE
Fixes false alarm event

### PR DESCRIPTION
**Pull request by Henri215**
## What Does This PR Do

Fixes false alarm event.
## Why It's Good For The Game

Bugs are bad
## Images of changes

image

## Testing

- Did include for testing purpose the false alarm event to the event_container
- Forced the event and got the results (image above)
## Changelog
:cl:
fix: False alarm event has been fixed
/:cl:

### CODE REVIEW

File: code/modules/events/event_container.dm [hunk 1]
```diff
@@ -127,6 +127,26 @@ GLOBAL_LIST_EMPTY(event_last_fired)
 	next_event = EM
 	return EM
 
+/datum/event_container/false_events // Used by the /datum/event/falsealarm event
+	available_events = list(
```
[the remark this answers is not in the document]
Comment (Henri215) [reply code/modules/events/event_container.dm:131]: It's not.

[the remark this answers is not in the document]
Comment (S34NW) [reply code/modules/events/event_container.dm:131]: Might want to make it a static

[the remark this answers is not in the document]
Comment (Henri215) [reply code/modules/events/event_container.dm]: Changed how it works, moved all events announcements into the false alarm file.

**Review by S34NW — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
Hardcoding all the values is kinda ehhhhhhhh but oh well

**Pull request merged**